13.technical-support/02.Adware/docs.en.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Adware/Viruses
+published: true
+taxonomy:
+    category:
+        - docs
+---
+
+If you suspect your computer is infected by malware or viruses, first of all you need to check your computer with free utilities from leading antiviruses.
+
+**Free antivirus utility**
+
```

Comment:
There are many of them and you can use any one you prefer, here is simply a couple examples: